<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admittance</class>
 <widget class="QDialog" name="Admittance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 	#333333</string>
  </property>
  <widget class="QPushButton" name="start_login_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>85</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yandex Sans Display Bold</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #0080ff;
border-radius: 12;
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_regisration_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>185</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yandex Sans Display Bold</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #0080ff;
border-radius: 12;
</string>
   </property>
   <property name="text">
    <string>Rregistration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yandex Sans Text Bold</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #333333</string>
   </property>
   <property name="text">
    <string>What you nead ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yandex Sans Text Bold</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #333333</string>
   </property>
   <property name="text">
    <string>or</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
